src/frontend/src/lib/i18n/en.json

```
@@ -606,7 +607,7 @@
 			"slippage_exceeded": "$oisy_short protects you: The trade is outside of your slippage tolerance of $maxSlippage%. Please adjust the max slippage and try again.",
 			"pool_not_found": "Swap failed. Pool not found.",
 			"deposit_error": "Swap failed. We couldn’t deposit your tokens into the pool. Please try again.",
-			"withdraw_failed": "Swap failed and withdraw also failed. Please check your unused balance and try manual withdrawal: $url",
+			"withdraw_failed": "Swap failed and withdraw also failed. Please check your unused balance and try manual withdrawal: ",
```

double-check if you need the whitespace at the end

```suggestion
			"withdraw_failed": "Swap failed and withdraw also failed. Please check your unused balance and try manual withdrawal:",
```

---

Yeap, it is needed here